src/main/resources/spring/database/schema/experiment-schema.sql

```diff
@@ -50,3 +50,9 @@ subTaskId int NOT NULL,
 PRIMARY KEY (taskId, subTaskId)
 );
 
+CREATE TABLE IF NOT EXISTS ExperimentTasks_AdditionalBlobResults (
+    taskId int NOT NULL,
+    resultType VARCHAR(20),
```

Comment:
Please have a look at the `ExperimentTasks_AdditionalResults` table. The `resultType` should be a `resultId` and it should also be part of the primary key.

Reply:
Refactored the implementation as per Suggestions.

- replaced `resultType` with `resultId`
- added `resultId` to primary key
- added `Enum ExerimentTaskBlobResultType` for getting resultId